scss/tech.scss

```diff
@@ -1,42 +1,26 @@
 $tech-logo-size: 100px;
 
 .techs-container {
-  display: flex;
-  flex-wrap: wrap;
-  margin-left: 100px;
+  display: grid;
+  margin: 0 auto;
+  width: 900px;
```

Comment:
Now that grid has a fixed width `grid-template-columns: repeat(3, 300px);` , try deleting this line